src/index.d.ts

```diff
@@ -9,6 +9,7 @@ export interface GoogleFontLoaderProps {
     fonts: Font[];
     subsets?: string[];
     display?: 'auto' | 'block' | 'swap' | 'fallback' | 'optional';
+    contentDocument?: Node;
```

Comment:
And same here - is this the right typescript type? What is `Node` in ts?

Reply:
Yep. Meant to come back and clean this up. Good catch. I'm new to typescript 😅 , and need to look this up.